components/ai_chat/core/browser/ai_chat_service.h

```diff
@@ -174,7 +174,8 @@ class AIChatService : public KeyedService,
 
  private:
   // Key is uuid
-  using ConversationMap = std::map<std::string, mojom::ConversationPtr>;
+  using ConversationMap =
+      std::map<std::string, mojom::ConversationPtr, std::less<>>;
```

Comment:
just curious, what does the `std::less<>` do here?

Reply:
allow the map to compare the string key with a string_view